### PR TITLE
OMG! Ubuntu! Presence

### PR DESCRIPTION
**Pull request by v30xy**
OMG! Ubuntu Presence. Here is some screenshot.
Ekran görüntüsü 2021-03-16 02-23-21
Ekran görüntüsü 2021-03-16 02-23-40
Ekran görüntüsü 2021-03-16 02-24-07
Ekran görüntüsü 2021-03-16 02-28-10

### CODE REVIEW

File: websites/O/OMG! Ubuntu!/presence.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        ?.lastChild.textContent.trim() || "Unknown Category";
+
+    presenceData.details = "Looking a category:";
+    presenceData.state = category;
+
+    OMGUbuntu.setActivity(presenceData);
+  } else if (page.includes("/page/")) {
+    const articlesPage = location.pathname.split("/")[2] || "Unknown Page";
+
+    presenceData.details = "Looking articles";
```
Comment (SlowLife1661) [inline websites/O/OMG! Ubuntu!/presence.ts:49]: This seems like a grammar error kekw

File: websites/O/OMG! Ubuntu!/presence.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    presenceData.state = tag;
+
+    OMGUbuntu.setActivity(presenceData);
+  } else if (page.includes("/category/")) {
+    const category =
+      document
+        .querySelector(".content-container .loop h2")
+        ?.lastChild.textContent.trim() || "Unknown Category";
+
+    presenceData.details = "Looking a category:";
```
Comment (SlowLife1661) [inline websites/O/OMG! Ubuntu!/presence.ts:41]: These 2 detail as well

File: websites/O/OMG! Ubuntu!/presence.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    OMGUbuntu.setActivity(presenceData);
+  } else if (page.includes("/") && articleHeader) {
+    presenceData.details = "Reading an article:";
+    presenceData.smallImageKey = "reading";
+    presenceData.state = articleHeader;
+    presenceData.buttons = [{ label: "Read Article", url: location.href }];
+
+    OMGUbuntu.setActivity(presenceData);
+  } else if (pages[page]) {
+    presenceData.details = "Looking a page:";
```
Comment (SlowLife1661) [inline websites/O/OMG! Ubuntu!/presence.ts:70]: ^

Comment (v30xy) [reply websites/O/OMG! Ubuntu!/presence.ts]: :ok_hand: Thanks

File: websites/O/OMG! Ubuntu!/presence.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  if (page.includes("/tag/")) {
+    const tag =
+      document
+        .querySelector(".content-container .loop h2")
+        ?.lastChild.textContent.trim() || "Unknown Tag";
+
+    presenceData.details = "Looking at a tag:";
+    presenceData.state = tag;
+
+    OMGUbuntu.setActivity(presenceData);
```
Comment (Bas950) [inline websites/O/OMG! Ubuntu!/presence.ts:34]: just use setACtivity once at the end

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/O/OMG! Ubuntu!/presence.ts (hunk 4 above)
Comment (v30xy) [reply websites/O/OMG! Ubuntu!/presence.ts]: :+1:

**Review by Bas950 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]
LGTM!

**Pull request merged**